flink-tests/src/test/java/org/apache/flink/test/recovery/ProcessFailureCancelingITCase.java

```diff
@@ -221,15 +210,11 @@ public Long map(Long value) throws Exception {
             final Collection<JobID> jobIds = waitForRunningJobs(clusterClient, timeout);
 
             assertThat(jobIds, hasSize(1));
-            final JobID jobId = jobIds.iterator().next();
 
             // kill the TaskManager after the job started to run
             taskManagerProcess.destroy();
             taskManagerProcess = null;
 
-            // try to cancel the job
-            clusterClient.cancel(jobId).get();
-
             // we should see a failure within reasonable time (10s is the ask timeout).
```

Comment:
```suggestion
            // we should see a failure within reasonable time (100s is the ask timeout).
```
The ask timeout is set to 100 seconds in this test.